hub/powertoys/peek.md

```diff
@@ -21,12 +20,33 @@ Using <kbd>Left</kbd> and <kbd>Right</kbd> or <kbd>Up</kbd> and <kbd>Down</kbd>,
 
 ## Pin preview window position and size
 
-The Peek window adjusts its size based on the dimensions of the images being previewed. However, if you prefer to keep the window's size and position, you can use the pinning feature. By selecting the pinning button, the window will preserve the current size and position. Selecting the pinning button again will unpin the window. When unpinned, the Peek window will return to the default position and size when previewing the next file.
+The Peek window adjusts its size based on the dimensions of the images being previewed. However, if you prefer to keep the window's size and position, you can use the pinning feature.
+By selecting the pinning button, the window will preserve the current size and position. Selecting the pinning button again will unpin the window. When unpinned, the Peek window will return to the default position and size when previewing the next file.
 
 ## Open file with the default program
 
 Select **Open with** or <kbd>Enter</kbd> to open the current file with the default program.
 
+## See extra information about the current file
+
+Hover over the preview to see extra information about the file, including its size, type, and when it was last modified.
+
+## Delete files
+
+Press the <kbd>Delete</kbd> key to move the current file to the Recycle Bin.
+
+By default, a confirmation dialog will appear before deletions. To skip future confirmations, either:
+
+- Check the "Don't ask me again" checkbox in the dialog.
+- Uncheck the "Ask for confirmation before deleting files" option in Peek's settings page.
+
+![Example of Peek's file deletion confirmation dialog](../images/pt-peek-delete-confirmation.png)
```

Comment:
Minor suggestion. I try to begin alt text with "A screenshot of" or "A picture of", etc:
```suggestion
![A screenshot of Peek's file deletion confirmation dialog](../images/pt-peek-delete-confirmation.png)
```